### DISCUSSION TITLE
The apply page for the Student Developer Pack without the ‘upload’ button

### DISCUSSION THREAD

**Discussion by ghost** [community/community#17779] [GitHub Education]
Hello, I want to enroll in the Student Pack program, but I cannot upload my student document. I can’t use the other option, the camera, because I don’t have it.

**Comment by deargh** [comment]
I had the same issue, in both devices I use, I couldn't fix it but I tried again a few days later and the upload button was there! If it's not urgent just wait a few days and it may be back!

**Reply by ghost** [reply]
I try everyday but it didn't work. :(

**Comment by mlarasusan** [comment]
Hello 👋

I have some information here that can help you [✨](https://emojipedia.org/sparkles/)

👉 Check [this](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/) to learn how to get the GitHub Student Developer Pack without a student ID
👉 Also, check the reviews of these students: [1](https://github.com/orgs/github-community/discussions/31#discussioncomment-2561585); [2](https://github.com/orgs/github-community/discussions/16); [3](https://github.com/orgs/github-community/discussions/65) & [4](https://github.com/orgs/github-community/discussions/67) - Looks like the last two had the same issue as you  
👉 And here is the [link](https://support.github.com/contact/education) to contact our support team
👉 Student Developer Pack Application & [FAQs](https://github.com/orgs/github-community/discussions/9)
